registry-client.js

```diff
@@ -115,16 +116,20 @@ this.Registry = (function (Promise, XMLHttpRequest, DOMParser, undefined) {
                           null
 
                         if (serviceURL) {
+                          found = true
                           resolve(serviceURL)
+                          break
                         }
                       }
                     }
                   }
                 }
               }
-              var errorMsg = `No service URL found for \nResource: ${resourceURI}\nStandard: ${standardURI}\nInterface: ${_interfaceURI}\nAuthType: ${authType}`
-              console.error(errorMsg)
-              reject(new Error(errorMsg))
+              if (found === false) {
+                var errorMsg = `No service URL found for \nResource: ${resourceURI}\nStandard: ${standardURI}\nInterface: ${_interfaceURI}\nAuthType: ${authType}`
```

Comment:
Unfortunately, IE 11, which is the current version at NRC, does not support the back-tick style string interpolation.  It will need to be switched back to concatenation.